test/ERC7498-Revert.t.sol

```diff
@@ -196,7 +196,7 @@ contract ERC7498_Revert is BaseRedeemablesTest {
         // campaignId: 1
         // requirementsIndex: 0
         // redemptionHash: bytes32(0)
-        bytes memory extraData = abi.encode(1, 0, bytes32(0));
+        bytes memory extraData = abi.encode(1, 0, bytes32(0), defaultTraitRedemptionTokenIds, uint256(0), bytes(""));
```

Comment:
in all these cases, update the comment above so it's easier to understand the values passed (the other option is to make them vars but this way is at least less pressure on stack)
you can also make the comment multiline so it doesn't take up so many lines, so e.g.
`campaignId: 1, requirementsIndex: 0, ` etc.